tutorials/haskell/mustache-templates/code/README.md

```diff
@@ -0,0 +1,9 @@
+# Image processing
+
+Code for the “image processing” tutorial.
```

Comment:
@mrkkrp This one is not for image processing. :) Also, line 9 says BSD but the license file is MIT.